src/som/primitives/processes/ChannelPrimitives.java

```diff
@@ -249,6 +261,18 @@ protected boolean isTaggedWithIgnoringEagerness(final Class<?> tag) {
         return super.isTaggedWithIgnoringEagerness(tag);
       }
     }
+
+    private static boolean stopOnRead() {
```

Comment:
I was planning to take care of these methods by actually integrating them into the breakpoint nodes. I think, it will be best to check the stepping strategy there.

The stepping strategy implementation itself is also something that probably can be solved simpler with a enum or so.